greynoise.json

```diff
@@ -101,7 +101,7 @@
                     "data_type": "string",
                     "example_values": [
                         "The IP has never been observed scanning the Internet",
-                        "The IP has been observed by the GreyNoise sensor network"
+                        "The IP has been observed by the test sensor network"
```

Comment:
@dhwanis-crest : this change should be reverted.  This should read "GreyNoise sensor network"

Reply:
Yes, we would prefer that we keep these as we've noted them.  Thank you.

Reply:
Done